.github/workflows/scripts/scanner-offline-bundle-version-matrix.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+#
+# Print the mapping of StackRox releases to scanner vulnerability bundle schema
+# versions. One entry per line, separated by whitespace.
+
+set -euo pipefail
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# GITHUB_URL_PATTERN points to vulnerability version file for a given branch/tag.
+GITHUB_URL_PATTERN="https://raw.githubusercontent.com/stackrox/stackrox/%s/scanner/VULNERABILITY_VERSION"
+
+RELEASE_VERSION_FILE="scanner/updater/version/RELEASE_VERSION"
```

Comment:
not sure if it's in-scope of this PR, but we should probably deprecate this file, right? Should we add something to the file like a comment (or change the name of the file) to make it clear it's deprecated?

Reply:
Unfortunately, we cannot. At least not with the current implementation. The file provides a list of releases that we should build offline bundles for. I can think of some way to achieve the same by inspecting the git repository, and hard-code some releases where V4 was not supported or bundles should be different, but for now I think the file is a smoother transition.